docs/content/Metrics.md

```diff
@@ -3,11 +3,168 @@ layout: doc_page
+|`*/errors`|Number of cache errors.||0|
+
+## Ingestion Metrics
+
+|Metric|Description|Dimensions|Normal Value|
+|------|-----------|----------|------------|
+|`ingest/events/thrownAway`|Number of events rejected because they are outside the windowPeriod.|dataSource.|0|
+|`ingest/events/unparseable`|Number of events rejected because the events are unparseable.|dataSource.|0|
+|`ingest/events/processed`|Number of events successfully processed.|dataSource.|Equal to your # of events.|
+|`ingest/rows/output`|Number of Druid rows persisted.|dataSource.|Your # of events with rollup.|
+|`ingest/persists/count`|Number of times persist occurred.|dataSource.|Depends on configuration.|
+|`ingest/persists/time`|Milliseconds spent doing intermediate persist.|dataSource.|Depends on configuration.|Generally a few minutes at most.|
+|`ingest/persists/backPressure`|Number of persists pending.|dataSource.|0|
+|`ingest/persists/failed`|Number of persists that failed.|dataSource.|0|
+|`ingest/handoff/failed`|Number of handoffs that failed.|dataSource.|0|
+
+### Indexing Service
+
+|Metric|Description|Dimensions|Normal Value|
+|------|-----------|----------|------------|
+|`task/run/time`|Milliseconds taken to run task.|dataSource, taskType, taskStatus.|Varies.|
+|`segment/added/bytes`|Size in bytes of new segments created.|dataSource, taskType, interval.|Varies.|
+|`segment/moved/bytes`|Size in bytes of segments moved/archived via the Move Task.|dataSource, taskType, interval.|Varies.|
+|`segment/nuked/bytes`|Size in bytes of segments deleted via the Kill Task.|dataSource, taskType, interval.|Varies.|
+
+## Coordination
+
+These metrics are for the Druid coordinator and are reset each time the coordinator runs the coordination logic.
+
+|Metric|Description|Dimensions|Normal Value|
+|------|-----------|----------|------------|
+|`segment/added/count`|Number of segments added to the cluster.|tier.|Varies.|
+|`segment/moved/count`|Number of segments moved in the cluster.|tier.|Varies.|
+|`segment/dropped/count`|Number of segments dropped due to being overshadowed.|tier.|Varies.|
+|`segment/deleted/count`|Number of segments dropped due to rules.|tier.|Varies.|
+|`segment/unneeded/count`|Number of segments dropped due to being marked as unused.|tier.|Varies.|
+|`segment/cost/raw`|Used in cost balancing. The raw cost of hosting segments.|tier.|Varies.|
+|`segment/cost/normalization`|Used in cost balancing. The normalization of hosting segments.|tier.|Varies.|
+|`segment/cost/normalized`|Used in cost balancing. The normalized cost of hosting segments.|tier.|Varies.|
+|`segment/loadQueue/size`|Size in bytes of segments to load.|server.|Varies.|
+|`segment/loadQueue/failed`|Number of segments that failed to load.|server.|0|
+|`segment/loadQueue/count`|Number of segments to load.|server.|Varies.|
+|`segment/dropQueue/count`|Number of segments to drop.|server.|Varies.|
+|`segment/size`|Maximum byte limit available for segments.|dataSource.|Varies.|
```

Comment:
himanshu: is this the "total byte size of segments" in the datasource?
fj: no, it is the maxSize configured for the historical node

himanshu: historical has maxSize configured globally (not per datasource level), how come does this metric has dataSource as a dimension then? also , code at https://github.com/druid-io/druid/pull/1208/files#diff-5494ef6d4623daa4fc35a5558f023294L216 seems to suggest its just total size of segments for the data source.